collections/dominode-resources/processing/legacy.py

```diff
@@ -0,0 +1,66 @@
+import typing
+
+from qgis.core import (
+    QgsProcessingAlgorithm,
+    QgsProcessingParameterExpression,
+    QgsProcessingOutputString,
+)
+from qgis.PyQt.QtCore import QCoreApplication
+
+
+class Legacy(QgsProcessingAlgorithm):
```

Comment:
Please choose a more descriptive name that signals that this is a validation step. Something like `NoopValidator` or similar. Also, please change the name of the file to reflect that, to be consistent with how the other algorithm files that exist on this repo are being named.